### PR TITLE
Adding TextToSpeechManager to PresentationActivity.

### PR DESCRIPTION
**Pull request by alaaalzibda**
After the DRUMROLL player is completed I call the speak function which is inside the textToSpeechManager

### CODE REVIEW

File: Random-Name-Picker.iml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/.gradle" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
```
Comment (Gear61) [inline Random-Name-Picker.iml:15]: Hmmm, I need to remove this from the repository. It's an IDE specific file.

File: app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingFragment.java [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
```
Comment (Gear61) [inline app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingFragment.java:5]: Please delete this, it's not being used anywhere.

File: app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingFragment.java [hunk 3]
```diff
@@ -148,7 +149,6 @@ public void choose() {
         if (nameChoosingAdapter.getCount() == 0) {
             return;
         }
-
```
Comment (Gear61) [inline app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingFragment.java:151]: nit: Please put this new line back?

**Review by Gear61 — CHANGES_REQUESTED** [review]
This is a good start, but we need to remove the old implementation of this app speaking feature. You need to remove the TextToSpeech instance from this activity and slot in to where it was being invoked before.

File: Random-Name-Picker.iml (hunk 1 above)
Comment (alaaalzibda) [reply Random-Name-Picker.iml:15]: Oh, I keep forgetting it, even in the new commit I forgot it, sorry
Do you add /Random-Name-Picker.iml inside the .gitignore file and then you call 
git add . as usual or git add . then git reset -- Random-name-picker.iml ?

**Pull request closed without merging**